app/_how-tos/throttle-apis-with-services-and-consumers.md

```diff
@@ -0,0 +1,156 @@
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## 1. Create privileged Consumers
+
+In this how to, we'll be creating two Consumers that have different rate limits from the limits applied to the Service. These Consumers act as a type of "privileged" Consumer in this scenario that have higher rate limits, but they will still be prevented from exceeding the Service rate limits. The reason for this is to prevent DDoS type attacks on your APIs. 
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: jsmith
+      keyauth_credentials:
+      - key: jsmith-key
+    - username: tsmith
+      keyauth_credentials:
+      - key: tsmith-key
+{% endentity_examples %}
+
+## 2. Enable authentication
+
+Authentication lets you identify a Consumer so that you can apply rate limiting to Consumers.
+This example uses the [Key Authentication](/plugins/key-auth/) plugin, but you can use any authentication plugin that you prefer.
+
+Enable the plugin globally, which means it applies to all {{site.base_gateway}} Services and Routes:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: key-auth
+      config:
+        key_names:
+          - apikey
+{% endentity_examples %}
+
+## 3. Enable rate limiting on Consumers with the Rate Limiting plugin 
+
+Enable the [Rate Limiting plugin](/plugins/rate-limiting/) for the Consumers. 
+In this example, the limit is 6 requests per minute per Consumer.
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: rate-limiting
+      consumer: jsmith
+      config:
+        minute: 6
+        limit_by: consumer
+        policy: local
+    - name: rate-limiting
+      consumer: tsmith
+      config:
+        minute: 6
+        limit_by: consumer
+        policy: local
+{% endentity_examples %}
+
+## 4. Enable rate limits on the Service with the Service Protection plugin
+
+Now we can apply a rate limit on the Service itself using the [Service Protection](/plugins/service-protection/) plugin. In this example, we are setting the limit to 10 requests per minute. Due to [plugin execution order](/gateway/plugin-execution-order/), the Service Protection plugin is applied *before* the Rate Limiting plugin. This means that if multiple Consumers are making requests at the same time, together they cannot make more than 10 requests per minute, even if the Consumer rate limit is 6 requests per minute. 
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: service-protection
+      service: example-service
+      config:
+        window_size: [60]
+        limit: [10]
+        namespace: service-protection-plugin
+{% endentity_examples %}
+
+## 5. Validate
+
+To test that the Service Protection plugin correctly applies rate limits to the Service, you'll run requests from both Consumers in quick succession. 
+
+First, run the following command to test the rate limiting as the `jsmith` Consumer:
+
+{% validation request-check %}
```

Comment:
There's a specific rate limit check validation, should use that one instead. Eg here: https://github.com/Kong/developer.konghq.com/blob/service-protection/app/_how-tos/get-started-with-gateway.md?plain=1#L365-L370